README.md

```diff
@@ -26,3 +26,11 @@ tests. All of these packages can be installed with the following commands::
 To test the package, execute the following command::
 
     $ nosetests uvtools/tests/test_dspec.py uvtools/tests/test_utils.py
+
+## Making a Release
+
+To make a release of `aipy` (both on Github and PyPI), head to the most current
```

Comment:
I think we're suffering from copy-pasta here

Reply:
Aye.